docs/deployment/self-hosted/kubernetes-self-hosted.mdx

```diff
@@ -11,16 +10,12 @@ keywords:
 
 import Thumbnail from "@site/src/components/Thumbnail";
 
-# Self-Hosted (Customer Managed) Data Plane Installation Guide
-
-:::info
+# Kubernetes Self-Hosted Data Plane Installation Guide
 
 Documentation here targets customers who want to self host and self manage their clusters as well as their workloads.
```

Comment:
I know this is brought over en-masse from the existing k8s page, but these aren't customers if they're self-hosting. Given that, I'll stop reviewing this page.